common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -27,7 +27,21 @@ run() {
 
 killall_program() {
     PROGRAM_NAME=$1
-    KILL_SIG=${2:-TERM}
+    KILL_SIG=${2:-KILL}
```

Comment:
If an explicit signal is given, it's probably for something like sending a SIGUSR1 or maybe SIGHUP that *doesn't* terminate the program. So the second kill simply shouldn't be done in that case.